action.yml

```
@@ -98,7 +98,7 @@ runs:
         git remote set-url origin https://${{ inputs.access-token }}@github.com/wandb/docs.git ; \
         git pull origin main
```

```suggestion
        git fetch origin
```
You don't need to update local `main`, you just need to fetch `origin` before doing the reset. This makes it more obvious what we are doing, IMO.